packages/cli/src/lib/commands/history-util.ts

```diff
@@ -132,6 +131,11 @@ export const getHistory = async (client: SmartThingsClient, limit: number, perRe
 	const items: DeviceActivity[] = [...history.items]
 	while (items.length < limit && history.hasNext()) {
 		await history.next()
+		// The API allows the user to continue to records before the specified after with paging so
```

Comment:
This wording is a little hard to follow. I think it's "continue to records before the specified after" that confuses me--maybe something like "continue to retrieve records before the specified 'after' date"?

Reply:
Confusing? That's an understatement. Mangled is more like it. 😄 I must have messed this in editing. I've updated it to:

```
		// The API allows the user to continue to view history from before the specified "after"
		// with paging so we stop processing if we get items that come before the specified "after".
```